src/binary/ibc-relayer/commands/start.ts

```diff
@@ -208,8 +208,16 @@ async function run(options: Options, logger: Logger) {
     throw new Error('dest chain not found in registry');
   }
 
-  const nodeA = await signingClient(srcChain, options.mnemonic, logger);
-  const nodeB = await signingClient(destChain, options.mnemonic, logger);
+  const nodeA = await signingClient(
+    srcChain,
+    options.mnemonic,
+    logger.child({ label: srcChain })
```

Comment:
Create a child logger with metadata overridden, `label` in this case. It results in:
```
verbose: [local_wasm] Get header for height ...
```

We can adjust how the `label` is displayed in [simple format function](https://github.com/confio/ts-relayer/pull/134/files#diff-1ec075639d7551edbb2e6eacfa20415bc8bf6896649951d95da8cd3dc2cf06e3R101). Should we move it before the colon?
```
verbose [local_wasm]: Get header for height ...
```

Or maybe can you think of something better-looking?

Reply:
I think the first version is fine.
Nice to see how this is added in.

Reply:
Hmmm... I assume you haven't tested this manually:

`const srcChain = chains[options.src];` is a `Chain` object. 

Either use `{label: options.src}` or `{label: srcChain.chainId}`

Reply:
You're right I didn't test it manually but I was going to before pulling out of the draft.

Anyway, it's fixed now, and additionally, I made the `logger.child` function type-safe: 2427a13

From now, `logger.child` options are restricted to `{ label: string }` while before we could pass anything and make mistakes.
```diff
-- child(options: Object): Logger
++ child(options: { label: string }): Logger
```

Reply:
You could extend it to `Record<string, string>` in the future as well, but I doubt we want complex objects in there.

Reply:
I see but for now, we have no other usage of child logger than with `label` so I'm going to keep it strict.